transaction-builder/src/lib.rs

```diff
@@ -1047,7 +1047,7 @@ impl TransactionBuilder {
     ///                             belonging to this key pair.
     ///  - `staker_key_pair`:       The key pair used to sign the incoming transaction. The staker
     ///                             address will be derived from this key pair.
-    ///  - `value`:                 The value of the stake to be inactivated. This is moved from the
+    ///  - `value`:                 The value of the stake to be activated. This is moved from the
     ///                             active balance of the staker.
```

Comment:
I feel this description is not great. Wasn't great either, so not a fault of this PR.

My suggestion:
```suggestion
    ///  - `value`:                 The portion of the total stake to be set as the active stake. Can be at most the total stake. The difference between this value and the total stake is set as inactive stake. To inactivate all stake, set this value to 0.
```